pkg/server/handler/handler.go

```diff
@@ -113,6 +114,10 @@ func (h *Handler) setCacheable(w http.ResponseWriter) {
 }
 */
 
+func (h *Handler) applicationBundleClient() *applicationbundle.Client {
+	return applicationbundle.NewClient(h.client, h.namespace)
```

Comment:
`h.namespace` is always the namespace you are running in e.g. `unikorn-kubernetes`.  I'm assuming this isn't what you want ultimately and there is going to be an encore?

Reply:
Yeahp -- this PR is just to clear up the theoretical problem of bundles coming from any namespace. That makes it possible to change it so that you can tell the controller to look in a namespace _other_ than the one the controller lives in -- but, fix the roof first, then put um, glitter on it.